tests/test_log.py

```diff
@@ -218,33 +218,13 @@ def print_mock(mocker):
 15: undefined method `cp_config' for
 16: An error has occurred, this and all later migrations canceled
 17: undefined method `import_data' for nil:NilClass (NoMethodError)
-18: undefined method `[]' for nil:NilClass (NoMethodError)
-## Full log:
-1: c75ff8b-2ee0-4a6a-a94e-, org=] ERROR  org.
-2: SEVERE: The web application [/gutterball] registered the JDBC driver
-3: NO problem on this line
-4: Exception in thread "Resource Destroyer in BasicResourcePool.close()"
-5: javax.persistence.PersistenceException: unexpected error when rollbacking
-6: Could not find the inverse association for repository
-7: PG::Error: ERROR: update or delete on table
-8: NoMethodError: undefined method
-9: usr/sbin/foreman-rake db:migrate returned 1 instead of one of [0]
-10: /usr/sbin/foreman-rake db:seed returned 1 instead of one of [0]
-11: [E] undefined method `finished' for nil:NilClass (NoMethodError)
-12: [ProxyAPI::ProxyException]: Unable to detect
-13: (ActiveModel::MissingAttributeError)
-14: ActionView::Template::Error (undefined method
-15: undefined method `cp_config' for
-16: An error has occurred, this and all later migrations canceled
-17: undefined method `import_data' for nil:NilClass (NoMethodError)
 18: undefined method `[]' for nil:NilClass (NoMethodError)'''
 
 CONTENT_WITH_SPECIAL_CHAR = 'éçã'.encode('utf-8')
 
 REPORT_WITH_SPECIAL_CHAR = '''### Analyzing /var/log/candlepin/candlepin.log:
 ## Errors found:
-## Full log:
-1: éçã'''
+None'''
```

Comment:
You can:
Either, remove None and Dont print anything
OR, Print some feedback on not finding anything. 

I prefer 2nd option which you are also ok with.